js/app.js

```diff
@@ -2,9 +2,8 @@ const term = document.querySelector('.term');
 const definition = document.querySelector(".definition");
 const checkButton = document.querySelector('.check');
 const nextButton = document.querySelector('.next');
-const audio = document.querySelector('.audio');
```

Comment:
Main issue here was that the class `.audio` referred to the `div` wrapping the `audio` element, which has no `.play` attribute - `.play()` needs to be called on the `audio` element itself.